examples/00-EDB/04_edb_parametrized_design.py

```diff
@@ -192,10 +181,8 @@
            ["$pcb_len", "-($ms_width + $ms_spacing)/2"],
           ],
          ]
-# ###############################################################################
-# Add traces to EDB
-# ~~~~~~~~~~~~~~~~~
-# Add traces to EDB.
+###############################################################################
+# Add traces to the EDB.
```

Comment:
I can't quite figure out when you are using periods and when you aren't. The previous formatting had a title and then descriptive text. Now, with just the single # sign, I'm not sure what we have.